protos/encodings.proto

```diff
@@ -263,6 +271,7 @@ message ArrayEncoding {
         FixedSizeBinary fixed_size_binary = 11;
         BitpackedForNonNeg bitpacked_for_non_neg = 12;
         Constant constant = 13;
+        Bitpack2 bitpack2 = 14;
```

Comment:
We don't need it for this PR but I'm thinking of putting the new 2.1 encodings in `message Compression { }` instead of `ArrayEncoding`.  On the plus side, we can probably name it `bitpack` and not `bitpack2` once we do that.  Let's save for future work though.

Reply:
gotcha, good idea.